### ISSUE TITLE
Older seedlots not updating new BV data

### ISSUE THREAD

**Issue by SLDonnelly** [bcgov/nr-spar#1667]
**Describe the Bug**
Breeding values were update for Sitka spruce (Ss), so when I open the record, put it in edit mode and click on the recalculate button it should update the new breeding values from the oracle API and then recalculate the GW traits

**Expected Behaviour**
When I open a record in review mode and click recalculate, new BV should be pulled and the new GW should be recalculated

**Actual Behaviour**
The system is spinning and not pulling data

![Image](https://github.com/user-attachments/assets/445792d9-829b-434d-9bf4-9b554c534ec4)

**Comment by SLDonnelly** [comment]
Opened PR for this request is here: [https://github.com/bcgov/nr-spar/pull/1669](https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fbcgov%2Fnr-spar%2Fpull%2F1669&data=05%7C02%7Csabina.donnelly%40gov.bc.ca%7C023b0a2091b743f7a7de08dcea1de00d%7C6fdb52003d0d4a8ab036d3685e359adc%7C0%7C0%7C638642661251365545%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=N%2BmAwAQeFHh2B4icCNsIygJIPGlHsqLo9IB6EXGvBCo%3D&reserved=0)

**Comment by SLDonnelly** [comment]
Previous changes in the PR didn't fix the issue. Ricardo found the following. @mgaseta would you be able to follow up on Ricardo's work and see if we can complete this issue?

here's the new issue:
![Image](https://github.com/user-attachments/assets/eaf787dd-ddbf-4a2d-8803-5bed66182474)

**Comment by mgaseta** [comment]
Sure! I'll get right into it.

**Comment by SLDonnelly** [comment]
Hi @mgaseta I've tested your PR. The recalculate section is now working and doesn't get stuck :) However it looks like the form isn't pulling in new breeding values. These are the testing steps that I took:

1. created a new seedlot and registered it
2. Approved the seedlots
3. Opened old SPAR to update the breeding values for Sitka Spruce trees in orchard 172
4. Reopened the seedlot registration form in SPAR2
5. In the review and edit view, I clicked on recalculted

The Expected result is that new breeding values are brought up (see several GVO values updated from 2 to 10)

https://github.com/user-attachments/assets/53ebbc42-7075-433d-8043-97b52da6ca42

![Image](https://github.com/user-attachments/assets/b209ba45-6242-47f2-adbf-b702dbf64e1d)
